WcaOnRails/app/jobs/sync_mailing_lists_job.rb

```diff
@@ -26,7 +26,7 @@ def perform
     GsuiteMailingLists.sync_group("results@worldcubeassociation.org", Team.wrt.current_members.includes(:user).map(&:user).map(&:email))
     GsuiteMailingLists.sync_group("software@worldcubeassociation.org", Team.wst.current_members.includes(:user).map(&:user).map(&:email))
     GsuiteMailingLists.sync_group("software-admin@worldcubeassociation.org", Team.wst_admin.current_members.includes(:user).map(&:user).map(&:email))
-    translator_users = UserGroup.translators.map(&:roles).flatten.map(&:user)
+    translator_users = UserGroup.translators.flat_map(&:users)
```

Comment:
I am still not 100% happy because types are not obvious in Ruby. Calling `UserGroup.translators`, I would expect the result to be **ONE** group that is called "Translators" and which reprensents all translators.

I know that we have decided to implement translators for different languages through different sub-groups, but for somebody who wasn't involved in that decision, this variable name `UserGroup.Translators` might not at all be obvious.

What about calling this `UserGroup.translator_groups`?

Reply:
`UserGroup.translator_groups` looks like a good idea. I guess this is applicable to some other group type functions as well. Since those are not relevant to this PR, I'll change it there once I touch that code for some other PR.

Reply:
Nitpick: Now you've changed the name below in the `UserGroup` class, but you haven't changed it here where you're calling it

Reply:
Sorry my bad. I've changed it now. Also rebased.